tests/test_connector.rs

```diff
@@ -585,3 +585,16 @@ fn test_ft_transfer_call_fee_greater_than_amount() {
     let balance = total_supply_eth(&master_account, CONTRACT_ACC);
     assert_eq!(balance, 0);
 }
+
+#[test]
+fn test_get_accounts_counter() {
+    let (master_account, contract) = init(CUSTODIAN_ADDRESS);
+    call_deposit_near(&contract, CONTRACT_ACC);
+
+    let counter = master_account.view(CONTRACT_ACC.into(), "get_accounts_counter", &[]);
+    let counter: u128 = String::from_utf8(counter.unwrap())
+        .unwrap()
+        .parse()
+        .unwrap();
+    assert_eq!(counter, 2);
```

Comment:
Why is the value 2? I only see one call to deposit so I would only expect there to be one account.

Reply:
Please look to that test:
```
#[test]
fn test_near_deposit_balance_total_supply() {
    let (master_account, contract) = init(CUSTODIAN_ADDRESS);
    call_deposit_near(&contract, CONTRACT_ACC);

    let balance = get_near_balance(&master_account, DEPOSITED_RECIPIENT, CONTRACT_ACC);
    assert_eq!(balance, DEPOSITED_AMOUNT - DEPOSITED_FEE);

    let balance = get_near_balance(&master_account, CONTRACT_ACC, CONTRACT_ACC);
    assert_eq!(balance, DEPOSITED_FEE);

```

Reply:
Right, there is a fee as well, so one deposit creates two accounts. Thanks for clarifying; it might be worth saying this in a comment as well.

Also, I recommend extending the test to have a transfer between those two accounts after the deposit, and confirm that the count does not change (because the number of unique accounts is the same).